src/MLJNaiveBayesInterface.jl

```diff
@@ -141,4 +141,125 @@ MMI.is_pure_julia(::Type{<:MultinomialNBClassifier}) = true
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Finite`; check the scitype with `schema(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above. Predictions are probabilistic but
+  uncalibrated.
+- `predict_mode(mach, Xnew)`: Return the mode of above predictions.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `c_counts`: A dictionary containing the observed count of each input class.
+- `c_stats`: A dictionary containing observed statistics on each input class. Each class is
+  represented by a `DataStats` object, with the following fields:
+    - `n_vars`: The number of variables used to describe the class's behavior.
+    - `n_obs`: The number of times the class is observed.
+    - `obs_axis`: The axis along which the observations were computed.
+- `gaussians`: A per class dictionary of gaussians, each representing the distribution of
+  the class.
+- `n_obs`: The total number of seen observations.
+
+# Examples
+
+```
+using MLJ
+GaussianNB = @load GaussianNBClassifier pkg=NaiveBayes
+
+X, y = @load_iris
+clf = GaussianNB(k=3)
+mach = machine(clf, X, y) |> fit!
+
+fitted_params(mach)
+
+preds = predict(mach, X);
+preds[1]
+predict_mode(mach, X)
+```
+
+See also
+[`MultinomialNBClassifier`](@ref)
+"""
+GaussianNBClassifier
+"""
+$(MMI.doc_header(MultinomialNBClassifier))
+
+
+`MultinomialNBClassifier`: MultinomialNBClassifier Naive Bayes classifier.
+Assumes variables have a multinomial distribution. Good for counts and text classification.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Count`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Finite`; check the scitype with `schema(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above. Predictions are probabilistic but
+  uncalibrated.
+- `predict_mode(mach, Xnew)`: Return the mode of above predictions.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `c_counts`: A dictionary containing the observed count of each input class.
+- `c_stats`: A dictionary containing observed statistics on each input class. Each class is
+  represented by a `DataStats` object, with the following fields:
+    - `n_vars`: The number of variables used to describe the class's behavior.
+    - `n_obs`: The number of times the class is observed.
+    - `obs_axis`: The axis along which the observations were computed.
+- `gaussians`: A per class dictionary of gaussians, each representing the distribution of
```

Comment:
multinomials here I guess?

Reply:
This is actually a typo here on my part, should be resolved on latest commit!